Makefile

```diff
@@ -0,0 +1,18 @@
+# Docker is the only dependency here
+# Build the dashboard image for host or cross-compile
+# This Makefile can only be run on a amd64 host
+
+build-image:
```

Comment:
Awesome! :) Let's start with a dockerfile with build environment. Can you maybe base the image on ubuntu (or some other standard distro) and just `sudo apt-get install java golang nodejs ...`? 

What about keeping this PR open for a while so that we can keep our high level discussions around?